core/src/main/java/org/millr/slick/servlets/item/EditItemServlet.java

```diff
@@ -82,7 +82,10 @@
 	
 	@Override
 	protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response) throws ServletException, IOException {
-		LOGGER.debug(">>>> Entering doPost");
+	// response.setContentType("text/html; charset=UTF-8");
+	// response.setCharacterEncoding("UTF-8");
+	request.setCharacterEncoding("UTF-8");
```

Comment:
Same.

Reply:
I did all changes as proposed - but I wonder if we need to change all occurrences of org.apache.commons.lang e.g. import org.apache.commons.lang.StringUtils to V3 as well ?